admin/src/Game/Board.tsx

```diff
@@ -0,0 +1,53 @@
+import { useRecordContext } from "react-admin";
+import {
+  Container,
+  Typography,
+  Table,
+  TableBody,
+  TableCell,
+} from "@material-ui/core";
+
+export const Board = (props: any) => {
+  const record = useRecordContext(props);
+  const board = record.board;
+
+  return (
+    <Container>
+      <Typography>Board</Typography>
+      <Table>
+        <TableBody>
+          {board.map((row: [], rowIndex: number) => (
+            <tr>
```

Comment:
why do not you use TableRow of material ui?

Reply:
For lack of time, when I use the MUI component, it does not have the right size and I had to go quickly to the next one